Assets/GothicVR/Scripts/Manager/SkyManager.cs

```diff
@@ -36,32 +51,79 @@ public void InitSky()
                 CreatePresetState(new SkyState(), (state) => state.PresetDay0())
             });
 
+
             RenderSettings.fog = true;
             RenderSettings.fogMode = FogMode.Linear;
             RenderSettings.ambientMode = AmbientMode.Flat;
-            Interpolate();
+            InitRainState();
+            noSky = true;
+
+            Interpolate(new DateTime());
         }
 
-        private void Interpolate(DateTime _)
+        private void UpdateStateTexAndFog(SkyState currentState)
         {
-            Interpolate();
+            if (SettingsManager.GameSettings.GothicINISettings.ContainsKey("SKY_OUTDOOR"))
+            {
+                var currentDay = GameTime.I.GetDay();
+                var day = (currentDay + 1);
+
+                // hacky way to use the proper color for the current day until animTex is implemented
```

Comment:
What's animTex? Please create a ticket and link this line to ensure we fix it soonish.

Reply:
animated textures, the next feature that i am working on, used by sky and will be used by water meshes and some particles
animated textures refers to all textures that end in A[0-9]-C.TEX and need to be grouped